custom_news_feed/sample_news/minimal_news.json

```diff
@@ -8,15 +8,19 @@
             "Text":"Based on fossil and biological evidence, most scientists accept that birds are a specialised subgroup of theropod dinosaurs and, more specifically, members of Maniraptora, a group of theropods which includes dromaeosaurids and oviraptorosaurs, among others. As scientists have discovered more theropods closely related to birds, the previously clear distinction between non-birds and birds has become blurred. Recent discoveries in the Liaoning Province of northeast China, which demonstrate many small theropod feathered dinosaurs, contribute to this ambiguity.",
             "LinkUrl":"",
             "LinkTitle": "",
-            "ImageUrl": ""
+            "ImageUrl": "",
+            "StartPublishingDate": "2023-05-05T06:00:00.000Z",
+            "EndPublishingDate": "2023-05-30T23:59:59.000Z"
```

Comment:
Q: ist es sinnvoll in den sample news dieses Enddatum zu setzen? So wird die Nachricht ja nie angezeigt, wenn jemand neu das Plugin runterlädt oder?

Reply:
ist korrigiert